nntrainer/layers/div_layer.cpp

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 SeungBaek Hong <sb92.hong@samsung.com>
+ *
+ * @file   div_layer.cpp
+ * @date   30 August 2024
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author SeungBaek Hong <sb92.hong@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  This is div layer class (operation layer)
+ *
+ */
+
+#include <div_layer.h>
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <node_exporter.h>
+#include <util_func.h>
+
+#include <layer_context.h>
+
+namespace nntrainer {
+
+static constexpr size_t SINGLE_INOUT_IDX = 0;
+
+void DivLayer::finalize(InitLayerContext &context) {
+  context.setOutputDimensions({context.getInputDimensions()[0]});
+}
+
+void DivLayer::forwarding(RunLayerContext &context, bool training) {
+  Tensor &hidden_ = context.getOutput(SINGLE_INOUT_IDX);
+
+  const Tensor &input0 = context.getInput(0);
+  const Tensor &input1 = context.getInput(1);
+
```

Comment:
Maybe we need to check the case by dividing by 0?

Reply:
I'm not sure.. Is checking zero-division error at layer level recommendable?
I mean, I tried checking them value-by-value at computation backend level, but it gives a bit of latency overhead during computation process.
Where do you think we should put them? or, do we really need them? (just curious)

Reply:
Yes......! your comment seems reasonable. It might cause too much latency; Since this is an atomic tensor operation, it would be better not to perform the value-by-value calculation. It seems more reasonable to let the user handle it :)